From 7aa828bb168f5179d19df067ad1813fdc5f58b13 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Wed, 31 Oct 2018 09:53:38 +0800
Subject: [PATCH 4992/5242] MLK-20136-02 driver: clk: imx: keep DRAM PLL
 always on for i.MX8MQ

commit  e59e354f65da476cd751a8c24ed67a4cc2cfc3af from
https://source.codeaurora.org/external/imx/linux-imx.git

Keep the DRAM PLL always on by default on i.MX8MQ.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8mq.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/imx/clk-imx8mq.c b/drivers/clk/imx/clk-imx8mq.c
index 4a988f1a..64492dfa 100644
--- a/drivers/clk/imx/clk-imx8mq.c
+++ b/drivers/clk/imx/clk-imx8mq.c
@@ -281,7 +281,7 @@
 	IMX8MQ_CLK_USB_BUS,
 	IMX8MQ_CLK_MAIN_AXI, IMX8MQ_CLK_A53_CG,
 	IMX8MQ_CLK_TMU_ROOT,
-	IMX8MQ_CLK_DRAM_APB,
+	IMX8MQ_CLK_DRAM_APB, IMX8MQ_DRAM_PLL_OUT,
 };
 
 static struct clk ** const uart_clks[] __initconst = {
-- 
1.7.9.5

